locales/el/el.json

```diff
@@ -1,618 +1,618 @@
-    "Delete Account": "Delete Account",
-    "Delete API Token": "Delete API Token",
-    "Delete File": "Delete File",
-    "Delete Resource": "Delete Resource",
-    "Delete Selected": "Delete Selected",
-    "Delete Team": "Delete Team",
-    "Denmark": "Denmark",
-    "Detach": "Detach",
-    "Detach Resource": "Detach Resource",
-    "Detach Selected": "Detach Selected",
-    "Details": "Details",
-    "Disable": "Disable",
-    "Djibouti": "Djibouti",
-    "Do you really want to leave? You have unsaved changes.": "Do you really want to leave? You have unsaved changes.",
-    "Dominica": "Dominica",
-    "Dominican Republic": "Dominican Republic",
-    "Done.": "Done.",
-    "Download": "Download",
+    "could not be found.": "δεν βρέθηκε.",
+    "Create": "Δημιουργήσετε",
+    "Create & Add Another": "Δημιουργία & Προσθήκη Άλλου",
+    "Create :resource": "Δημιουργία :resource",
+    "Create a new team to collaborate with others on projects.": "Δημιουργήστε μια νέα ομάδα για να συνεργαστείτε με άλλους σε έργα.",
+    "Create Account": "Δημιουργία Λογαριασμού",
+    "Create API Token": "Δημιουργία Token API",
+    "Create New Team": "Δημιουργία Νέας Ομάδας",
+    "Create Team": "Δημιουργία Ομάδας",
+    "Created.": "Δημιουργήθηκε.",
+    "Croatia": "Κροατία",
+    "Cuba": "Κούβα",
+    "Curaçao": "Κουρασάο",
+    "Current Password": "Τρέχων Κωδικός Πρόσβασης",
+    "Customize": "Προσαρμογή",
+    "Cyprus": "Κύπρος",
+    "Czech Republic": "Τσεχία",
+    "Dashboard": "Πίνακας",
+    "December": "Δεκεμβρίου",
+    "Decrease": "Μειώσετε",
+    "Delete": "Διαγράψετε",
+    "Delete Account": "Διαγραφή Λογαριασμού",
+    "Delete API Token": "Διαγραφή Token API",
+    "Delete File": "Διαγραφή Αρχείου",
+    "Delete Resource": "Διαγραφή Πόρου",
+    "Delete Selected": "Διαγραφή Επιλεγμένων",
+    "Delete Team": "Διαγραφή Ομάδας",
+    "Denmark": "Δανία",
+    "Detach": "Αποσυνδέσετε",
+    "Detach Resource": "Αποσύνδεση Πόρου",
+    "Detach Selected": "Αποσύνδεση Επιλεγμένων",
+    "Details": "Στοιχεία",
+    "Disable": "Απενεργοποιήσετε",
+    "Djibouti": "Τζιμπουτί",
+    "Do you really want to leave? You have unsaved changes.": "Θέλεις πραγματικά να φύγεις; Έχετε μη αποθηκευμένες αλλαγές.",
+    "Dominica": "Κυριακή",
+    "Dominican Republic": "Δομινικανή Δημοκρατία",
+    "Done.": "Γίνεται.",
+    "Download": "Λήψη",
     "E-Mail Address": "Διεύθυνση E-Mail",
-    "Ecuador": "Ecuador",
-    "Edit": "Edit",
-    "Edit :resource": "Edit :resource",
-    "Edit Attached": "Edit Attached",
-    "Editor": "Editor",
-    "Editor users have the ability to read, create, and update.": "Editor users have the ability to read, create, and update.",
-    "Egypt": "Egypt",
-    "El Salvador": "El Salvador",
-    "Email": "Email",
-    "Email Address": "Email Address",
+    "Ecuador": "Εκουαδόρ",
+    "Edit": "Επεξεργασία",
+    "Edit :resource": "Επεξεργασία :resource",
+    "Edit Attached": "Επεξεργασία Συνημμένου",
+    "Editor": "Επεξεργασία",
```

Comment:
An editor in this case is referring to a person with the capacity to edit something within some organization (like a publisher, or auditor, etc.). When I first saw it I thought it meant something like a "WYSIWYG" editor, but reading the next string I realized it had a different meaning. So the Greek string should be "Συντάκτης" .